resources/path_collector/path_collector_make_condor.py

```diff
@@ -64,7 +71,7 @@ def make_condor_file(  # pylint: disable=R0913,R0914
 +FileSystemDomain = "blah"
 should_transfer_files = YES
 transfer_input_files = {",".join([os.path.abspath(f) for f in transfer_input_files])}
-request_cpus = {cpus}
+request_cpus = {cpus}{accounting_group_attr}
```

Comment:
This line just looks wrong by eye.  I know you have a newline in the string itself, but I'd recommend putting this on its own line.  Condor ignores empty lines, so it should be fine.